mava/types.py

```diff
@@ -96,6 +96,20 @@ class PPOTransition(NamedTuple):
     info: Dict
 
 
+class RNNPPOTransition(NamedTuple):
+    """Transition tuple for PPO."""
+
+    done: Done
+    action: Action
+    value: Value
+    reward: chex.Array
+    log_prob: chex.Array
+    obs: chex.Array
+    policy_hidden_state: HiddenState
+    critic_hidden_state: HiddenState
```

Comment:
I think this should be `HiddenStates`, HiddenStates above this so you can use it here

Reply:
Nice catch!